src/mgr/MgrCommands.h

```diff
@@ -100,10 +100,13 @@ COMMAND("osd test-reweight-by-pg " \
 
 COMMAND("osd scrub " \
 	"name=who,type=CephString", \
-	"initiate scrub on osd <who>", "osd", "rw", "cli,rest")
+	"initiate scrub on osd <who>, or use <all|any|*> to scrub all", \
```

Comment:
why shall we add "all|any", it this a convention we need to follow? if yes, could you point me any of the places where we are following it?